recipes/tensorboard/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set version = "0.1.6" %}
+{% set build_number = "0" %}
+
+package:
+  name: tensorboard
+  version: {{ version }}
+
+build:
+  # There are two wheels (py2, py3) in pypi, that seem equivalent to me.
+  # So for the time being, we try building a noarch package.
+  # Also see:
+  #  https://github.com/conda-forge/conda-smithy/pull/531
+  noarch: python
+  skip: True  # [not linux]
+  number: {{ build_number }}
+  entry_points:
+    - tensorboard = tensorboard.main:main
+  script:
+    - pip install --no-deps tensorflow-tensorboard=={{ version }}
+
+requirements:
+  # See: https://pypi.python.org/pypi/tensorflow-tensorboard/0.1.5
+  build:
+    - pip
+  run:
+    - bleach ==1.5.0
+    - html5lib ==0.9999999
```

Comment:
Do these need to be this tight?

Reply:
They are this tight both in the pypi package and in our current tensorflow recipe. I do not really know if they need to be that tight.